enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 - Multiple audit policies can be defined in a hosted cluster, independent of the management cluster policies.
   See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
-- Allow some forwarding in the _management_ cluster to be configured from the _hosted_ cluster.
-- Keep direct control of management cluster forwarding on the management cluster.
-  - hosted cluster has no direct access to control plane configuration.
-  - data plane resources can request forwarding by the management cluster.
-  - management cluster HLO automatically validates data-plane requests, and enables them if they are safe.
+- Data plane (customer) requests audit log forwarding by modifying data-plane CLF configuration, forwarding from control plane starts automatically.
 
 ### Non-Goals
 
 - This proposal only covers API audit logs. No change to application, infrastructure or node audit logs.
-- Management cluster is not _required_ to honour all possible forwarding configurations allowed by CLF. \
 
 ## Proposal
 
 ### Workflow Description
 
 #### Hosted Cluster, data plane
 
-1. Create a `Policy.audit.k8s.io` resource with the desired policy
-   (see [api-audit-log-policy](/api-audit-log-policy.md))
-1. Create a `HypershiftLogForwarder` (HLF) resource with an `audit` input linked to the policy.
-1. Create HLF `outputs` and `pipelines` to forward the resulting logs.
-1. Optionally create a `ClusterLogForwarder` to forward other logs (not API-audit) as usual.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` with a HTTP intupt.
+ - Receiver should be identified with a special input name e.g. `hypershiftAPIAudit`
+ - Pipelines from the receiver may include an [api-audit-log-policy](/api-audit-log-policy.md) filter.
+1. Forward logs to any outputs as desired.
 
 #### Management Cluster, control plane
 
-On the _management_ cluster
-1. Install the Cluster Logging Operator (CLO), watching all name-spaces.
-1. Install the new Hypershift Logging Operator (HLO), watching all name-spaces.
-1. In each new HCP name-space, create a `ClusterLogForwarderTemplate` resource. \
-   This resource may be identical for all HCPs or customized per HCP.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` for each HCP to forward to Splunk and management-side CloudWatch.
+1. Install the new Hypershift Logging Operator (HLO) watching each HCP namespace.
+   - The HLO watches CLF configurations _in the data plane_, not the control plane.
+   - When the HLO detects a _control plane_ CLF requesting audit log forwarding, it updates
+     the _data plane_ CLF for that HCP to forward audit logs via a HTTP output to the _control plane_ CLF.
+   - The HLO should copy audit log policies from  _control plane_ CLF pipelines to the _data plane_,
+     to forward only those logs that will also be forwarded by the data plane CLF.
```

Comment:
Will there be a default set of policies applied if no policy is configured on the data plane?

Reply:
The control plane can impose such a policy itself, in addition to the data plane policy. You can apply multiple audit policies in the CLF filter chain.
The CLF audit policy has some documented default behavior (inherited from the old splunk audit exporter) but it is possible for the user to over-ride that and ask for 'literally everything' - I think the controle plane probably should impose it's own baseline policy to prevent indiscriminate "give me request bodies for everything" even if the user filter allows it.

Reply:
Control plane forcing a policy is something that won't fit for all use cases, as long as it can be managed per output then it will be OK.